### ISSUE TITLE
Move across words

### ISSUE THREAD

**Issue by nacho-rodriguez** [richrace/highlight-selected#125]
Like searching in Firefox/Chrome, I think it will be good idea if we could move across all the selected words with key F3 (for instance)
Ok, I may have 38 highlighted words, but I my file is too big I have to scroll a lot.

Thanks.

**Comment by k1-hedayati** [comment]
Yes this feature is really necessary, I used this feature alot in emacs to navigate across the file.
  
https://github.com/nschum/highlight-symbol.el

**Comment by gnbl** [comment]
I would like this, too.

**Comment by oztalha** [comment]
Find next is `command+g` in Atom.
Select the word and `command+g` to jump to the next occurrence.

**Comment by gnbl** [comment]
Find And Replace: Find Next Selected: CTRL+F3 in Windows
